src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--30 ">
+      <div class="calendar__day calendar__day--thu"></div>
```

Comment:
It works well but I think it will be better to set modifier for whole calendar for block, cause it change the look not only for the current one.